tests/e2e/actions.go

```diff
@@ -171,6 +172,17 @@ func (tr TestRun) startChain(
 		chain:     action.chain,
 		validator: action.validators[0].id,
 	}, verbose)
+
+	// store the fact that we started the chain
+	if tr.runningChains == nil {
```

Comment:
nit imo a more common pattern for this initialization would be a constructor for `TestRun` that initializes the map

Reply:
I put this into a seperate TestRun.initialize function. My problem with the constructor approach is that go doesn't support named arguments for funcs but for struct creation named arguments work. The calls where the TestRuns are constructed become huge blocks of 10 arguments which are only identifiable by their ordering if they go through a constructor, which I found worse than this solution